pkg/pillar/types/global.go

```diff
@@ -866,6 +868,7 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	configItemSpecMap.AddIntItem(NetworkDialTimeout, 10, 0, 3600)
 	configItemSpecMap.AddIntItem(LocationCloudInterval, HourInSec, 5*MinuteInSec, 0xFFFFFFFF)
 	configItemSpecMap.AddIntItem(LocationAppInterval, 20, 5, HourInSec)
+	configItemSpecMap.AddIntItem(NTPSourcesInterval, 10*MinuteInSec, MinuteInSec, 30*MinuteInSec)
```

Comment:
@rouming here you set the max value to 30 mins. I'm just wondering if it's a hard max like e.g. in the case of the metrics (because they are used to signal to the controller that the device is still online) or if it's just an arbitrary value. Because I don't think that it's a big deal if the controller is not informed about some small changes like the precision of some NTP sources

Reply:
Ah, I misread your previous message and missed "max". That's correct line indeed, and the range is 10 min - 30 min. Well, of course that's not a big deal, but reasoning was better to inform sooner rather than later so maximum was chosen having some reasonable time span. If you want to change it having some new requirements in mind - go ahead, personally I do not have any objections :)